src/ImageFeatures.jl

```diff
@@ -2,4 +2,29 @@ module ImageFeatures
 
 # package code goes here
 
+using Images, ColorTypes, FixedPointNumbers
+
+include("glcm.jl")
+
+import ColorTypes: U8, U16, Gray
```

Comment:
Can't you just say `using ColorTypes`? This line suggests your actual intent is to *extend* `U8`, `U16`, and `Gray`. (`import` allows you to extend them, whereas `using` does not).